examples/UserAPI.hs

```diff
@@ -5,7 +5,7 @@
 {-# language OverloadedStrings #-}
 {-# language TypeApplications #-}
 {-# language TypeOperators #-}
-module UserAPI where
```

Comment:
Was getting this error before making this change:
```
Building executable 'user-example' for servant-to-elm-0.4.2.0..
[1 of 2] Compiling Paths_servant_to_elm
[2 of 2] Compiling UserAPI

<no location info>: error:
    output was redirected with -o, but no output will be generated
because there is no Main module.
```